### PR TITLE
MICROAPP-14265 Zoom - maintenance update

### PR DESCRIPTION
**Pull request by LubosTomandl**

### CODE REVIEW

File: bundles/http/Citrix/Zoom-Meetings/e40510b3-a760-43b4-ab16-56572dc6303b.js [hunk 1]
```diff
@@ -281,7 +283,7 @@ async function incSyncMeetings(dataStore,client,context,latestSynchronizationTim
         next_page_token = json.next_page_token ?? "";
         const data = json.meetings;
         let meetingsData = data.filter(meeting => {
-            if (moment(meeting.created_at).isAfter(latestSynchronizationTime)) {
+            if (moment(meeting.created_at).isAfter(latestSynchronizationTime) && (meeting.type == 2 || meeting.type == 8) ) {
```
Comment (ondrejkralik) [inline bundles/http/Citrix/Zoom-Meetings/e40510b3-a760-43b4-ab16-56572dc6303b.js:286]: Nit: Personally, I would switch conditions just for small improvement (we don't have to create a moment instance).
But not a big issue.

```suggestion
            if ((meeting.type == 2 || meeting.type == 8) && moment(meeting.created_at).isAfter(latestSynchronizationTime)) {
```

**Review by ondrejkralik — APPROVED** [review]

**Comment by LubosTomandl** [comment]
@ondrejkralik  is it ok now?

File: bundles/http/Citrix/Zoom-Meetings/e40510b3-a760-43b4-ab16-56572dc6303b.js (hunk 1 above)
Comment (Ajcitrite) [reply bundles/http/Citrix/Zoom-Meetings/e40510b3-a760-43b4-ab16-56572dc6303b.js]: Change has been implemented.

Comment (ondrejkralik) [reply bundles/http/Citrix/Zoom-Meetings/e40510b3-a760-43b4-ab16-56572dc6303b.js]: 👍🏼

**Review by MichaelaVitusova — APPROVED** [review]

**Review by PetrSplichal — APPROVED** [review]

**Pull request merged**